volatility3/framework/plugins/windows/malfind.py

```diff
@@ -141,16 +141,30 @@ def _generator(self, procs):
         # determine if we're on a 32 or 64 bit kernel
         kernel = self.context.modules[self.config["kernel"]]
 
+        # set refined criteria to know when to add to "Notes" column
+        refined_criteria = [b"MZ", b"\x55\x8B", b"\x55\x48", b"\x55\x89"]
+
         is_32bit_arch = not symbols.symbol_table_is_64bit(
             self.context, kernel.symbol_table_name
         )
 
         for proc in procs:
+            # by default, "Notes" column will be set to none
+            notes = "None"
```

Comment:
Rather than a textual value, it may be worthwhile making this something derived from `BaseAbsentValue`, which might allow a future interface to filter our non-responses more easily?